staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -123,6 +123,12 @@ const (
 	//
 	// Enables managing request concurrency with prioritization and fairness at each server
 	RequestManagement featuregate.Feature = "RequestManagement"
+
+	// owner: @denkensk
+	// alpha: v1.15
+	//
+	// Enables NonPreempting option for priorityClass and pod.
+	NonPreemptingPriority featuregate.Feature = "NonPreemptingPriority"
```

Comment:
Drop the changes in this file, since this feature doesn't belong in `k8s.io/apiserver`. It is specific to pods, priorityclasses, and kube-scheduler